build.gradle

```diff
@@ -33,6 +33,12 @@ plugins {
   id "org.sonarqube" version "2.7"
 }
 
+repositories {
+  maven {
+    url "https://oss.sonatype.org/content/repositories/comgooglezetasql-1013"
```

Comment:
This is an unreleased repository. How do our clients get this dependency in a maven project without this?

Reply:
Don't tell anyone, running with Apache Beam is the current release verification step for ZetaSQL. Once this PR is passing, I'll publish the ZetaSQL release, delete this line and send it out for review.